tests/storage/migration.go

```diff
@@ -1237,7 +1227,7 @@ func createSmallImageForDestinationMigration(vm *virtv1.VirtualMachine, name, si
 	virtCli := kubevirt.Client()
 	vmi, err := virtCli.VirtualMachineInstance(vm.Namespace).Get(context.Background(), vm.Name, metav1.GetOptions{})
 	Expect(err).ShouldNot(HaveOccurred())
-	libstorage.CreateFSPVC(name, vmi.Namespace, size, nil)
+	libstorage.CreateFSPVC(name, vmi.Namespace, size, libstorage.WithStorageProfile())
```

Comment:
I think exact size is of significance here

Reply:
After checking the test path further the size of the PVC is of no relevance, the disk.img gets created by the test with a certain size.